_api_app/app/Http/routes.php

```diff
@@ -10,80 +10,83 @@
 | and give it the Closure to call when that URI is requested.
 |
  */
+$app->group(['namespace' => 'App\Http\Controllers'], function () use ($app) {
```

Comment:
Why are these groups removed? Does the new version of Lumen not support them anymore?

Reply:
they still support it but they change how the namespace and prefix work so that the group will create a url that can't be resolve. like the error i got was 404 not found because the namespace was cumulate in such a way that you find yourself with an unknown url

Reply:
OK